lib/honeybadger/plugins/active_job.rb

```diff
@@ -4,27 +4,42 @@ module ActiveJob
       # Ignore inline and test adapters, as well as the adapters that we support with their own plugins
       EXCLUDED_ADAPTERS = %i[inline test delayed_job faktory karafka resque shoryuken sidekiq sucker_punch].freeze
 
-      Plugin.register {
-        requirement { defined?(::Rails.application) && ::Rails.application }
-        requirement {
-          ::Rails.application.config.respond_to?(:active_job) &&
+      class << self
+        def perform_around(job, block)
+          Honeybadger.clear!
+          context = context(job)
+          block.call
+        rescue StandardError => e
+          Honeybadger.notify(e, context:)
+          raise e
+        end
+
+        def context(job) # rubocop:disable Metrics/MethodLength
+          {
+            arguments: job.arguments,
+            component: :good_job,
```

Comment:
Since we support a variety of adapters, it doesn't make sense to sent the component to `:good_job`.

```suggestion
```

Reply:
Good call. That was a vestigial bit left over from what inspired this PR.

What do you think about setting component to the job class similar to sidekiq?

Reply:
I think that makes sense — I probably should have done that already. :)

Reply:
fixed in [c2de915](https://github.com/honeybadger-io/honeybadger-ruby/pull/528/commits/c2de915b8d8703875d955da51d8d8035b1190151)